focus-android/app/src/androidTest/assets/same-site-cookies.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE HTML>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<!-- Page source copied from test.senglehardt.com/test/dfpi/set_storage_simple.html-->
```

Comment:
I would just clone this asset and use https://github.com/mozilla/anti-tracking-test-pages/blob/main/tracker-test/set_storage_simple.html

Reply:
thanks Aaron, done.